airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -503,7 +423,7 @@ def is_sorted_descending(self) -> bool:
         """
         Depending if there any state we read stream in ascending or descending order.
         """
-        return self._first_read
+        return not self._first_read
```

Comment:
I don't understand why? Did we have an error connected to this? Did we send the wrong `direction` parameter or what?

Reply:
Yes it was sending the wrong direction. https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-github/source_github/streams.py#L495-L496 states we want to sort in ascending order for the first run, then descending order for subsequent runs to allow the incremental behavior in https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-github/source_github/streams.py#L701-L702. However, the current stream version is setting `is_sorted_descending` to true if `self._first_read` is true, which is the opposite behavior.